src/content/docs/infrastructure/host-integrations/host-integrations-list/ibmmq-monitoring-integration.mdx

```diff
@@ -55,22 +57,32 @@ Our integration is compatible with IBM MQ 9.0 or higher.
 
 * Linux <img style={{ width: '32px', height: '32px'}} class="inline" title="Linux" alt="Linux" src={OsLinux}/>
 
-For a comprehensive list of specific Linux versions, check the table of [compatible operating systems](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent/#operating-systems).
+* Windows <img style={{ width: '32px', height: '32px'}} class="inline" title="Windows" alt="Windows" src={OsWindows}/>
+
+For a comprehensive list of specific Linux and Windows versions, check the table of [compatible operating systems](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent/#operating-systems).
+
+<Callout variant="important">
+  ARM64 architecture is not supported.
+</Callout>
 
 ### System requirements [#system-reqs]
 
 * [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic), minimum version 1.24.0.
-* Linux distribution [compatible with the infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
+* Linux distribution or Windows version [compatible with the infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
 * [IBM MQ Redistributable Client libraries](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/messaging/mqdev/redist/) on the agent box.
 * Check or add the libraries path in the `LD_LIBRARY_PATH` environment variable.
```

Comment:
LD_LIBRARY_PATH is only for Linux. For Windows we could link this document which explains how to add it https://www.ibm.com/docs/en/ibm-mq/9.1?topic=windows-testing-communication-between-client-server

Would suggest something like:
Check or add the libraries path in the `LD_LIBRARY_PATH` environment variable in Linux or follow [these steps](https://www.ibm.com/docs/en/ibm-mq/9.1?topic=windows-testing-communication-between-client-server) on Windows.

Reply:
Since you set it up locally if you have it in mind can you add a line clarifying it for windows?

You can propose it as a suggestion, or push directly to my branch, I'll give you access

Reply:
Pushed